### ISSUE TITLE
Adding custom token from a private network (Geth) 

### ISSUE THREAD

**Issue by alexacostarua** [ethereumproject/explorer#164]
Where would I update the drop-down menu.  And also is the" Verify and Publish Contract Source Code" necessary for my token to be added properly?

**Comment by mikeyb** [comment]
move mouse to search field at top of github.  search for `bitether` in this repo.  Make sure it is set to the `Code` tab on left  This issue tracker is for bugs, please do not use for support until you have exhausted all your resources, especially the simple ones that take 3 seconds.

**Comment by alexacostarua** [comment]
@mikeyb Look I'm just trying to get this to work with the most basic ERC20 contract out there...  I can clearly see my balance of custom tokens in metamask, can transfer them, but can't view the balances of my accounts with the coin, or the transactions through this explorer.  I've tried adding the address in all caps, no luck.  I've edited the drop-down menu as well (even though it doesn't matter as far as functionality).  Does this issue or bug have to do with Geth, because I could easily run this with my parity node if that's what it takes (although I prefer using Geth).   And also I really appreciate all the help you've been giving me, If you have a doge address for donations please PM or just message me.  Thanks.

**Comment by mikeyb** [comment]
Im not sure of the specifics of the code in place to interact with contracts in this code base as I have not been involved in development.  But keep in mind we all do this on a volunteer basis, have lives, jobs, families and the likes.  So if you are asking for help, help us by giving as much detail as possible.  And only ask for help after you have spent the time looking into the issue you are having thoroughly.

I should always know what operating system you are running, application versions, errors, what you have tried, what the configurations look like, etc.  This specific issue was just changing a template file, which I know you didnt take the time to research as it is quite obvious when you do.  Apologies if my unwillingness to help seems harsh, but we are not here to hold hands and walk every person who uses this software through a custom setup.

If @hackmod can jump in and explain any of the specifics about integrating other contracts that would be appreciated.  And @alexacostarua if you get this all working as you want, writing up a wiki doc would be highly appreciated for future users.

**Comment by mikeyb** [comment]
> https://github.com/ethereumproject/explorer/issues/161 - previous issue

**Comment by alexacostarua** [comment]
Thanks @mikeyb appreciate the input.  But don't forget I opened this issue and titled it this because in the other "issue" after posting my log of the explorer along with my OS, and config info.  You told me "Sounds like you didnt update the dropdown menu" and then closed the issue.  In my eyes it was a bit misleading because it seemed like you were inferring if I fixed the drop-down menu, then the issue would be resolved.  Thats the only reason I titled this that.  FYI this is the only explorer with supposable (Geth) ERC20 support.  And I can write up a wikidoc tutorial once I figure this out, it would not just be a breakthrough for me, but for the entire community.
